Elevel/Application/Features/QuestionCommands/UpdateQuestionCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿using AutoMapper;
+using Elevel.Application.Interfaces;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.QuestionCommands
+{
+    public class UpdateQuestionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid? Id { get; set; }
+            public string NameQuestion { get; set; }
+            public DateTimeOffset CreationDate { get; set; }
+            public bool Deleted { get; set; }
+            public Guid AnswerId { get; set; }
+            public Guid? AuditionId { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+            public async Task<Response> Handle(Request request, CancellationToken cancelationtoken)
+            {
+                var question = await _context.Questions.FirstOrDefaultAsync(a => a.NameQuestion == request.NameQuestion && a.Deleted == request.Deleted && a.CreationDate == request.CreationDate && a.AnswerId == request.AnswerId && a.AuditionId == request.AuditionId, cancelationtoken);
```

Comment:
you should get your entity by id, not by all props, it will be very slow sql query if do so

Reply:
I dunno why I've to do that. Maybe for more intense checking-up, but your variant is much less time-consuming, so changed, thx